analysis/model/src/main/kotlin/de/maibornwolff/codecharta/serialization/CompressedStreamHandler.kt

```diff
@@ -11,7 +11,8 @@ object CompressedStreamHandler {
         var content = input
         if (!input.markSupported()) { content = BufferedInputStream(input) }
         content.mark(2)
-        val byteArray = content.readNBytes(2)
+        val byteArray = ByteArray(2)
+        content.readNBytes(byteArray, 0, 2)
```

Comment:
Would you please Check, if we can use content.read() instead since it is supported by Java8 also. I think I just saw, that readNBytes is not available in Java8.

Reply:
Addressed in d35f7cb 👍